test/x86/avx2.c

```diff
@@ -9824,6 +9926,92 @@ test_simde_mm256_srai_epi32(SIMDE_MUNIT_TEST_ARGS) {
   return 0;
 }
 
+static int
+test_simde_mm_srav_epi32 (SIMDE_MUNIT_TEST_ARGS) {
+  static const struct {
+    const int32_t a[4];
+    const int32_t b[4];
+    const int32_t r[4];
+  } test_vec[] = {
+    { {  INT32_C(   956480390), -INT32_C(   327735143), -INT32_C(   355390379), -INT32_C(   386052259) },
+      { -INT32_C(  1743728675), -INT32_C(  1123534065), -INT32_C(   600872973),  INT32_C(  1413968589) },
+      {  INT32_C(           0), -INT32_C(           1), -INT32_C(           1), -INT32_C(           1) } },
```

Comment:
You're only getting 0 and 1 for results because you're always shifting by absurd values so you basically only get the highest bit.  The shift values should be 5 bits here.

Reply:
oh, I didn't think about this, I have redone the tests, hopefully they are correct this time.
native-aliases test is failing but it is unrelated to this PR.
Thank You